components/core/Text.tsx

```diff
@@ -44,3 +50,24 @@ export const LinkPrimaryVariant = styled(Link)`
   font-size: 16px;
   color: white;
 `;
+
+export function LinkifyText({ children }: LinkifyProps): ReactElement {
+  return (
+    <Linkify
+      componentDecorator={(decoratedHref, decoratedText, key) => (
+        <a
+          target="blank"
+          href={decoratedHref}
+          key={key}
+          style={{
+            color: `${theme.colors.brand.blue}`,
+          }}
+        >
+          {decoratedText}
+        </a>
```

Comment:
Does using a Chakra Link work? Maybe something like

```javascript
        <Link
          isExternal
          href={decoratedHref}
          key={key}
          color={theme.colors.brand.blue}
          _hover={{ color: theme.colors.brand.dark_teal }}
        >
          {decoratedText}
        </Link>
```

If that doesn't work, consider using `target="_blank"` (with an underscore, though it seems to work ok as is) and a `rel="noopener noreferrer"`

Reply:
Let me try this.

Reply:
That works! Should I use LinkPrimary or just Link?

Reply:
Hmm personally I think bold font weight is a bit much for these links, so just Link with the extra color + hover properties would do!

Reply:
Sounds good!